### PR TITLE
Handle case where all input fastq files to FastQCJob are zero_length.

### PR DESCRIPTION
**Pull request by charles-cowart**

### CODE REVIEW

File: sequence_processing_pipeline/FastQCJob.py [hunk 1]
```diff
@@ -83,6 +83,9 @@ def _find_projects(self, path_to_run_id_data_fastq_dir, is_raw_input):
             # extract only fastq files from the list
             files = [x for x in files if x.endswith('.fastq.gz')]
 
+            # remove all files in 'zero_files' sub-folder
+            files = [x for x in files if 'zero_files' not in x]
```
Comment (antgonza) [inline sequence_processing_pipeline/FastQCJob.py:87]: Out of curiosity, why split in two vs. just doing in one loop?

**Review by antgonza — COMMENTED** [review]
Looks good, only one comment. BTW has this been tested locally or in qiita-rc to make sure the code works as expected?

File: sequence_processing_pipeline/FastQCJob.py (hunk 1 above)
Comment (charles-cowart) [reply sequence_processing_pipeline/FastQCJob.py:87]: mainly for readability. Let me see if I can compound the if clause - don't think I've ever tried to use and inside a list comprehension before.

**Comment by charles-cowart** [comment]
@antgonza I planned to try it out on qiita-rc after you took a look at it. I'll try it out on rc now.

**Comment by charles-cowart** [comment]
@antgonza Tested on qiita-rc and works as intended. I've attached before and after images of the error displayed by Qiita. Note that the before image was taken from an older production Qiita job while the after image was taken from a job run on Qiita-rc. The new error message is much more informative and definitive. With a message as definitive as this one, we can dispense with the stack-trace, as that is always hard to read. I will create an issue for iterating on the error reporting.
before
after

**Pull request merged**